packages/state-manager/src/sagas/socket/startConnection/startConnection.saga.ts

```diff
@@ -136,24 +139,6 @@ export function subscribe(socket: Socket) {
 
     // Community
 
-    socket.on(SocketActionTypes.COMMUNITY_CREATED, async (payload: ResponseCreateCommunityPayload) => {
-      log(`${SocketActionTypes.COMMUNITY_CREATED}: ${payload}`)
-      // We can also set community metadata when we register the
-      // owner's certificate. I think the only issue is that we
-      // register the owner's certificate before initializing the
-      // community and thus the storage service.
-      emit(communitiesActions.sendCommunityMetadata())
-      emit(publicChannelsActions.createGeneralChannel())
-      // We also save the owner's CSR after registering their
-      // certificate. It works, but it might make more sense to get
-      // all the backend services up and running and then save the
-      // CSR, register the owner's certificate and set community
-      // metadata.
-      emit(identityActions.saveUserCsr())
```

Comment:
Since we are using an acknowledgement for the CREATE_COMMUNITY event, this was moved into createCommunitySaga